README.md

```diff
@@ -1,24 +1,20 @@
 # THREDDS Test Environment
 
-A packer + ansible project to build and deploy an Amazon Machine Image and Docker Container for use by the various THREDDS projects at Unidata for running automated tests.
-The AMI is used by our Jenkins server to launch worker nodes for running tests.
-The Docker Image will be used as the basis for a custom GitHub action for testing the various THREDDS projects.
-Both image types are based off the latest available image, AMI or Docker, for `ubuntu 20.04`.
-The AMI and docker images produced by packer are both called "thredds-test-environment".
+A packer + ansible project to build Docker images for use by the various THREDDS projects at Unidata for running automated tests.
+The Docker images are used as the basis for Jenkins build nodes as well as a custom GitHub action for testing the various THREDDS projects.
+The image is based off the latest available image for `ubuntu 20.04`.
```

Comment:
Still 20.04?

Reply:
Yeah, that will be the next step. I wanted to make sure I could get what we have working with docker Jenkins before biting off the jump from 20.04 -> 24.04 :-)